static/panes.js

```diff
@@ -117,6 +117,12 @@ function Pane(id) {
   cloneButton.innerHTML = 'o';
   cloneButton.title = 'disconnect';
 
+  var saveButton = document.createElement('a');
+  saveButton.innerHTML = '&#10515;';
```

Comment:
It'd be nice to have something easier to read. The arrow is rather thin and hard to tell apart.